travel_platform/izitravel_t.py

```diff
@@ -0,0 +1,84 @@
+# coding=utf-8
```

Comment:
Traceback (most recent call last):
  File "/Users/kateryna/PycharmProjects/itea_project/itea-advanced-python/travel_platform/izitravel_tests.py", line 17, in setUp
    'api.izi.travel', 'API_KEY'))
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/ConfigParser.py", line 607, in get
    raise NoSectionError(section)
NoSectionError: No section: 'api.izi.travel'